habitat_baselines/common/policy_registry.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+r"""PolicyRegistry is extended from habitat.Registry to provide
+registration for policy, while keeping Registry in habitat core intact.
+
+Import the policy registry object using
+
+``from habitat_baselines.common.policy_registry import policy_registry``
+
+Decorator for registry different kind of classes with unique keys
+
+- Register a policy: ``@policy_registry.register_policy``
+"""
+
+from typing import Optional
+
+from habitat.core.registry import Registry
+
+
+class PolicyRegistry(Registry):
+    @classmethod
+    def register_policy(cls, to_register=None, *, name: Optional[str] = None):
+        r"""Register a RL policy to registry with key 'name'.
```

Comment:
The doc strings are unfortunately not in the right format for our documentation system. Please see the `:param:` syntax we use elsewhere.

Reply:
Thanks!
Will fix that
I formatted docstring the same way as in the [BaselineRegistry](https://github.com/facebookresearch/habitat-api/blob/7f2121e1a31d304b5e897c8a97929de44b4f02aa/habitat_baselines/common/baseline_registry.py#L29).

Reply:
Actually, you might be right that we may not build docs for habitat_baselines, or habitat-api. Perhaps we should consider doing so if we don't.

Reply:
@Skylion007  Updated!

Reply:
@Skylion007 would you mind resolving this change request?
I've already updated code according to your comments